packages/profile-sync-controller/src/shared/storage-schema.ts

```diff
@@ -9,31 +9,33 @@ import { createSHA256Hash } from './encryption';
  */
 const ALLOW_ARBITRARY_KEYS = 'ALLOW_ARBITRARY_KEYS' as const;
 
+export enum UserStorageFeatureNames {
+  Notifications = 'notifications',
+  Accounts = 'accounts_v2',
+  Networks = 'networks',
+}
+
 export const USER_STORAGE_SCHEMA = {
-  notifications: ['notification_settings'],
-  accounts: [ALLOW_ARBITRARY_KEYS], // keyed by account addresses
-  networks: [ALLOW_ARBITRARY_KEYS], // keyed by chains/networks
+  [UserStorageFeatureNames.Notifications]: ['notification_settings'],
+  [UserStorageFeatureNames.Accounts]: [ALLOW_ARBITRARY_KEYS], // keyed by account addresses
+  [UserStorageFeatureNames.Networks]: [ALLOW_ARBITRARY_KEYS], // keyed by chains/networks
 } as const;
 
 type UserStorageSchema = typeof USER_STORAGE_SCHEMA;
 
-export type UserStorageFeatures = keyof UserStorageSchema;
-export type UserStorageFeatureKeys<Feature extends UserStorageFeatures> =
+export type UserStorageFeatureKeys<Feature extends UserStorageFeatureNames> =
   UserStorageSchema[Feature][0] extends typeof ALLOW_ARBITRARY_KEYS
     ? string
     : UserStorageSchema[Feature][number];
 
 type UserStorageFeatureAndKey = {
-  feature: UserStorageFeatures;
-  key: UserStorageFeatureKeys<UserStorageFeatures>;
+  feature: UserStorageFeatureNames;
```

Comment:
I think it would be nice if some outside interfaces are backwards compatible. Supporting string unions or enums.

E.g. if I'm an SDK user, I now have to add an additional import for using user storage. That's not great.

Reply:
Also can you make sure that we have the correct exports for the subpaths?

E.g. If I'm importing the SDK through /SDK, can I also access the enum from here?

Reply:
I've pushed a new update, making types retro compatible. Also ditched the enum for a plain old object!

So now every param could either use `USER_STORAGE_FEATURE_NAMES.xxxx` or `'notifications' | 'accounts_v2' | 'networks' | ...`